crates/sncast/README.md

```diff
@@ -209,7 +209,7 @@ response_raw: [0x6]
 
 ## Development
 
-Refer to [documentation](https://foundry-rs.github.io/starknet-foundry/development/environment-setup.html) to make sure you have all the pre-requisites, and to obtain an information on how to help to develop `sncast`.
+Refer to [documentation](https://foundry-rs.github.io/starknet-foundry/) to make sure you have all the pre-requisites, and to obtain an information on how to help to develop `sncast`.
```

Comment:
This is intentional, it points to section with the setup needed for development, so I don't think we should change it 😄